src/frontend/komponenter/Fagsak/TilkjentYtelse/BeregningDetalj.tsx

```diff
@@ -0,0 +1,88 @@
+import * as React from 'react';
+import { IBeregningDetalj, ytelsetype } from '../../../typer/beregning';
+import { FamilieIkonVelger } from '@navikt/familie-ikoner';
+import { formaterBeløp, hentAlder } from '../../../utils/formatter';
+import { Normaltekst } from 'nav-frontend-typografi';
+import { PersonType } from '../../../typer/person';
+import classNames from 'classnames';
+
+interface IProps {
+    beregningDetaljer: IBeregningDetalj[];
+}
+
+interface IDetaljRadProps {
+    detalj: IBeregningDetalj;
+    skalVisePersonalia: boolean;
+    skalViseStipletLinje: boolean;
+}
+
+const BeregningDetalj: React.FunctionComponent<IProps> = ({ beregningDetaljer }) => {
+    const detaljerForSøker = beregningDetaljer
+        .filter(detalj => detalj.person.type === PersonType.SØKER)
+        .map((detalj, index, alle) => {
+            return (
+                <DetaljRad
+                    detalj={detalj}
+                    skalVisePersonalia={index === 0}
+                    skalViseStipletLinje={index === alle.length - 1}
+                    key={index}
+                />
+            );
+        });
+    return (
+        <>
+            {detaljerForSøker}
+            {beregningDetaljer
+                .filter(detalj => detalj.person.type !== PersonType.SØKER)
+                .map((detalj, index) => {
+                    return (
+                        <DetaljRad
+                            detalj={detalj}
+                            skalVisePersonalia={true}
+                            skalViseStipletLinje={false}
+                            key={index}
+                        />
+                    );
+                })}
+        </>
+    );
+};
+
+const DetaljRad: React.FunctionComponent<IDetaljRadProps> = ({
+    detalj,
+    skalVisePersonalia,
+    skalViseStipletLinje,
+}) => {
+    const alder = hentAlder(detalj.person.fødselsdato);
+    const kolonneClassnames = classNames('tilkjentytelse-detaljer-kolonne', {
+        'familierelasjon-separator': skalViseStipletLinje,
+    });
+    return (
+        <div className="tilkjentytelse-detaljer-rad">
```

Comment:
Noen spesiell grunn til at dere bruker divs istedet for table? Ville nok prøvd å bruke nav-frontend sin tabell-css pakke for dette.

Reply:
I all hovedsak fordi man skal ha ekspandering/skjuling av detaljer i tabellene - og det har jeg forsøkt på tidligere med `table`-elementer - det blir fort ganske grisete med mange `tbody`-elementer som må holdes styr på. I så måte er `flexbox` med divs semantisk lettere å holde gyldig.

Reply:
Ja, jeg ser poenget. Jeg har tidligere gjort det slik at hver rad får en id og når den klikkes så aktiveres det en ekspander toggle på den raden som gjør at det rendres en ekstra "spesial-rad": https://github.com/navikt/dsop-kontroll-frontend/blob/05292676893f3ec2e9cd56419710402c2a598a2d/src/app/components/Okonomi/Innhenting/Konti/Konti.tsx#L231. 

Det viktigste er at vi bevarer tabell uu funksjonalitet.